shared/src/enums/exception-message.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum ExceptionMessage {
+    INVALID_EMAIL = 'Invalid email',
+    INVALID_PASSWORD = 'Invalid Password',
```

Comment:
You shouldn't let the user know which data is wrong.
Use the same invalid data message for both cases.
Example: Invalid data

Reply:
Resolved in [d34575e](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/48/commits/d34575e7dd23ca227683111e6ecb3c46d1653c89)